### PR TITLE
Add SIG Storage Spotlight

### PR DESCRIPTION
**Pull request by fsmunoz**
SIG-Storage Spotlight, done within the context of SIG-ContribEx Comms.

/cc @kaslin @xing-yang

### CODE REVIEW

**Comment by sftim** [comment]
(nit)
/retitle Add SIG Storage Spotlight

**Comment by sftim** [comment]
/hold

Spotlight on SIG blogs start at https://github.com/kubernetes/contributor-site and then once the text there is approved and published, we _mirror_ those articles to the main Kubernetes blog (and we set a canonical URL).

Is there already a PR against https://github.com/kubernetes/contributor-site that would put an article in https://k8s.dev/blog ?

**Comment by fsmunoz** [comment]
Thank you for the explanation and guiding me through this, I was unaware of that @sftim. I've opened https://github.com/kubernetes/contributor-site/pull/319 , hope this is the right way.

File: content/en/blog/_posts/sig-storage-spotlight.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: blog
+title: "SIG-Storage Spotlight"
```
Comment (sftim) [inline content/en/blog/_posts/sig-storage-spotlight.md:3]:
```suggestion
title: "Spotlight on SIG Storage"
```

**Review by sftim — COMMENTED** [review]
The filename for this should match other files already committed.

How about changing `content/en/blog/_posts/sig-storage-spotlight.md` to `content/en/blog/_posts/2022-08-22-sig-storage-spotlight.md`?

Also, please set a `canonicalUrl` in the front matter - see `content/en/blog/_posts/2022-08-02-sig-docs-spotlight/index.md` for an example.

File: content/en/blog/_posts/sig-storage-spotlight.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+slug: sig-storage-spotlight
+---
+
```
Comment (sftim) [inline content/en/blog/_posts/sig-storage-spotlight.md:6]:
```suggestion

**Author**: Frederico Muñoz (SAS)

```

**Comment by fsmunoz** [comment]
@sftim commited your suggestions, changed the file name and added `canonicalUrl`

**Comment by sftim** [comment]
/lgtm
/approve

https://github.com/kubernetes/website/pull/35084#issuecomment-1186492757 explains when we'd unhold.

File: content/en/blog/_posts/2022-08-22-sig-storage-spotlight.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+layout: blog
+title: "Spotlight on SIG Storage"
+slug: sig-storage-spotlight
+date: 2022-08-22
+canonicalUrl: https://kubernetes.dev/blog/2022/08/22/sig-docs-spotlight-2022/
```
Comment (sftim) [inline content/en/blog/_posts/2022-08-22-sig-storage-spotlight.md:6]: Oops
```suggestion
canonicalUrl: https://www.kubernetes.dev/blog/2022/08/22/sig-storage-spotlight-2022/
```

**Comment by sftim** [comment]
/hold cancel
/lgtm

[Upstream article](https://www.kubernetes.dev/blog/2022/08/22/sig-storage-spotlight-2022/) is published.

**Comment by fsmunoz** [comment]
Oops, thanks @sftim , I guess I followed the example a bit to much 😅

**Pull request merged**